server/src/main/java/org/opensearch/cluster/service/ClusterManagerTaskThrottler.java

```diff
@@ -185,6 +189,27 @@ public void onBeginSubmit(List<? extends TaskBatcher.BatchedTask> tasks) {
         });
     }
 
+    /**
+     * If throttling thresholds are set via static setting, it will update the threshold map.
+     * It may start throwing throttling exception to older nodes in cluster.
+     *
+     * On submission of first task, it validates if all nodes are of 2.5.0 or not and
+     * update startThrottling flag.
+     * Once the flag is set it will start throttling, and won't perform check for next tasks.
+     */
+    private boolean checkForThrottling(Long threshold, Long count, int size) {
```

Comment:
shouldThrottle as it is returning boolean

Reply:
Renamed it.